app/src/server/mod.rs

```diff
@@ -89,6 +91,7 @@ pub fn start(
         let name = manifest.project.name.clone().unwrap_or_else(|| "Ambient".into());
         server_world.add_components(server_world.resource_entity(), Entity::new().with(project_name(), name)).unwrap();
 
+        Entity::new().with(synced_resources(), ()).with(abs_asset_url(), format!("http://{public_host}:{http_interface_port}/content/")).spawn(&mut server_world);
```

Comment:
Hm so the `ProxyBaseUrlKey` seems to be lost now; the asset::url method used to use both but now it's just using ServerBaseUrl